sqlglot/dialects/snowflake.py

```diff
@@ -1087,3 +1087,12 @@ def strtotime_sql(self, expression: exp.StrToTime):
             return self.func(
                 f"{safe_prefix}TO_TIMESTAMP", expression.this, self.format_time(expression)
             )
+
+        def timestampsub_sql(self, expression: exp.TimestampSub):
+            return self.sql(
+                exp.TimestampAdd(
+                    this=expression.this,
+                    expression=expression.expression * -1,
```

Comment:
Do we need to wrap `expression.expression` in a `Paren` here?

Reply:
This is already handled by `__mul__` afaict:


```python
>>> sql = "SELECT TIMESTAMP_SUB(TIMESTAMP "2008-12-25 15:30:00+00", INTERVAL 2 + 1 MINUTE)"
>>> sqlglot.parse_one(sql, dialect="bigquery").sql("snowflake")
SELECT TIMESTAMPADD(MINUTE, (2 + 1) * -1, CAST('2008-12-25 15:30:00+00' AS TIMESTAMPTZ))
```